tests/scripts/task_build_adreno_bins.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -e
+set -u
+
+output_directory=$(realpath ${PWD}/build-adreno-target)
+rm -rf ${output_directory}
+
+mkdir -p ${output_directory}
+cd ${output_directory}
+
+cp ../cmake/config.cmake .
+
+echo set\(USE_CLML ON\) >> config.cmake
```

Comment:
Are these settings different than the ones in `task_config_build_adreno.sh`?

Reply:
Yes, we build target components (tvm_rpc & libtvm_runtime.so with CLML enable) here.